stylus/components/popover.styl

```diff
@@ -30,7 +30,7 @@ $popover
     button
     [role=button]
         display          block
-        padding          em(8px) em(32px) em(8px) em(42px)
+        padding          em(8px, 16px) em(32px, 16px) em(8px, 16px) em(42px, 16px)
```

Comment:
I wonder if we could use rem here ?

Reply:
Indeed, that would in fact be a way better solution. I forgot about this, thanks.